qa/L0_savedmodel_shape/gen_qa_models_savedmodel.py

```diff
@@ -0,0 +1,328 @@
+# Copyright (c) 2018-2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
This file should move to qa/common/gen_qa_noshape_models.py and be invoked by gen_qa_model_repository as we do with the other qa models. You will need to run it on a system with TitanV (or v100) and then copy over to dbcluster with the other qa model repositories.

Reply:
Already generated and put on the dbcluster. Will add to the gen_qa_model_repository and in qa/common/gen_qa_noshape_models.py